### ISSUE TITLE
No error page showing up if no error message is supplied

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1107]
**TL;DR** Currently, hitting student account creation rate limit - and any other errors in student/ teacher registration where we're not supplying an error message - is silent.

We knew we needed a clearer message for when teachers hit the account creation rate limit (see https://github.com/LLK/scratchr2/issues/4174), but we expected them to run into a page like "Something went wrong" as the heading, and then in bold "There was an error while processing your registration" in the body but no details at the end of registration. However, they actually aren't seeing that page at all - they are just getting "stuck" on the last step. 
@rschamp said:
> we give an "err" but no "msg".

@rschamp & I were discussing the difficulty in distinguishing between the error states users are reporting with teacher/ student accounts. To address that, @rschamp is planning to include the error code in the fallback error message.

**Comment by mewtaylor** [comment]
@rschamp can this be closed from #1110?

**Comment by rschamp** [comment]
Oh yup, thanks.
